spec/package-deps-spec.coffee

```diff
@@ -88,3 +88,13 @@ describe 'Package-Deps', ->
           expect(atom.packages.activatePackage.callCount).toBe(3)
           expect(view.show.callCount).toBe(1)
           expect(view.advance.callCount).toBe(2)
+
+  describe 'helpers', ->
+
+    describe 'guessName', ->
+      Helpers = require('../lib/helpers')
```

Comment:
I am afraid you are mistaken, `guessName` function doesn't need anything on the file system. It accepts path string and tries to guess package name from that string. The following tests are to ensure that it can extract a name from not just nix paths but from windows ones as well. The username I provided don't have to be on the file system, they are meta values and these tests should be passes for all of us (They even pass on Travis).

I hope it clarifies your concerns, if it does not, please tell me so I can explain it further :)